connector/src/test/scala/com/basho/riak/spark/rdd/timeseries/NotAvailableFeaturesTest.scala

```diff
@@ -0,0 +1,54 @@
+package com.basho.riak.spark.rdd.timeseries
+
+import com.basho.riak.client.core.netty.RiakResponseException
+import com.basho.riak.spark._
+import com.basho.riak.spark.rdd.AbstractRDDTest
+import org.apache.spark.SparkException
+import org.apache.spark.sql.Row
+import org.hamcrest.CustomTypeSafeMatcher
+import org.junit.rules.ExpectedException
+import org.junit.{Assume, Before, Rule, Test}
+
+class NotAvailableFeaturesTest extends AbstractRDDTest {
```

Comment:
There should be a brief description of this Test suite and instruction how it might be triggered